app/src/androidTest/java/de/test/antennapod/ui/PreferencesTest.java

```diff
@@ -80,7 +80,7 @@ public void testSwitchTheme() {
         }
         clickPreference(R.string.user_interface_label);
         clickPreference(R.string.pref_set_theme_title);
-        onView(withText(otherTheme)).perform(click());
+        clickPreference(otherTheme);
```

Comment:
Be careful. This is not a preference but an option in a dialog.

Reply:
thanks @ByteHamster , still learning. fixed. 🤞 that the tests will pass